railties/test/commands/notes_test.rb

```diff
@@ -8,51 +8,51 @@ class Rails::Command::NotesTest < ActiveSupport::TestCase
   setup :build_app
   teardown :teardown_app
 
-  test "`rails notes` displays results for default directories and default annotations" do
+  test "`rails notes` displays results for default directories and default annotations with aligned line number and annotation tag" do
     app_file "app/controllers/some_controller.rb", "# OPTIMIZE: note in app directory"
     app_file "config/initializers/some_initializer.rb", "# TODO: note in config directory"
     app_file "db/some_seeds.rb", "# FIXME: note in db directory"
     app_file "lib/some_file.rb", "# TODO: note in lib directory"
-    app_file "test/some_test.rb", 1000.times.map { "" }.join("\n") << "# FIXME: note in test directory"
+    app_file "test/some_test.rb", "\n" * 100 + "# FIXME: note in test directory"
```

Comment:
I kinda want us to define a private method that does

```ruby
def 💯
  100
end
```

just for the hell of it, but maybe not 😂